lcls-twincat-motion/Library/Tests/FB_StatePMPSEnables_Test.TcPOU

```diff
@@ -31,8 +31,7 @@ VAR
 END_VAR
 ]]></Declaration>
     <Implementation>
-      <ST><![CDATA[
-astPositionState[nGoalState].fPosition := 10;
+      <ST><![CDATA[astPositionState[nGoalState].fPosition := 10;
```

Comment:
The start of file and end of file newlines are intentional to make the code more readable in review here, you don't have to include them but you probably shouldn't be removing them.

Reply:
Oh okay I didn't know that was the reasoning. I will fix.